docs/_writing_dependencies_conda.rst

```diff
@@ -0,0 +1,318 @@
+        PREFIX=/Users/john/miniconda2
+        installing: python-3.5.2-0 ...
+        installing: conda-env-2.6.0-0 ...
+        installing: openssl-1.0.2j-0 ...
+        installing: pycosat-0.6.1-py35_1 ...
+        installing: readline-6.2-2 ...
+        installing: requests-2.11.1-py35_0 ...
+        installing: ruamel_yaml-0.11.14-py35_0 ...
+        installing: sqlite-3.13.0-0 ...
+        installing: tk-8.5.18-0 ...
+        installing: xz-5.2.2-0 ...
+        installing: yaml-0.1.6-0 ...
+        installing: zlib-1.2.8-3 ...
+        installing: conda-4.2.12-py35_0 ...
+        installing: pycrypto-2.6.1-py35_4 ...
+        installing: pip-8.1.2-py35_0 ...
+        installing: wheel-0.29.0-py35_0 ...
+        installing: setuptools-27.2.0-py35_0 ...
+        Python 3.5.2 :: Continuum Analytics, Inc.
+        creating default environment...
+        installation finished.
+        Fetching package metadata .......
+        Solving package specifications: ..........
+
+        Package plan for installation in environment /Users/john/miniconda2:
+
+        The following packages will be downloaded:
+
+            package                    |            build
+            ---------------------------|-----------------
+            conda-4.2.13               |           py35_0         389 KB
+
+        The following packages will be UPDATED:
+
+            conda: 4.2.12-py35_0 --> 4.2.13-py35_0
+
+        Conda installation succeeded - Conda is available at '/Users/john/miniconda2/bin/conda'
+
+While Galaxy can be configured to resolve dependencies various ways, Planemo
+is configured with opinionated defaults geared at making building tools that
+target Conda_ as easy as possible.
+
+During the introductory tool development tutorial, we called ``planemo tool_init``
+with the argument ``--requirement seqtk@1.2`` and the resulting tool contained
+the XML::
+
+    <requirements>
+        <requirement type="package" version="1.2">seqtk</requirement>
+    </requirements>
+
+As configured by Planemo, when Galaxy encounters these ``requirement`` tags it
+will attempt to install Conda, check for referenced packages (such as
+``seqtk``), and install them as needed for tool testing.
+
+We can check if the requirements on a tool are available in best practice
+Conda channels using an extended form of the ``planemo lint`` command. Passing
+``--conda_requirements`` flag will ensure all listed requirements are found.
+
+::
+
+    $ planemo lint --conda_requirements seqtk_seq.xml
+    Linting tool /Users/john/workspace/planemo/docs/writing/seqtk_seq_v6.xml
+      ...
+    Applying linter requirements_in_conda... CHECK
+    .. INFO: Requirement [seqtk@1.2] matches target in best practice Conda channel [bioconda].
```

Comment:
I guess we should make a separate issue if you want to discuss this more - I feel like a conda_check command or something would be reasonable - but it isn't want I was trying to capture with ``lint``. Does that make sense?